tree/dataframe/inc/ROOT/RDFInterfaceUtils.hxx

```diff
@@ -451,6 +451,18 @@ ColumnNames_t FindUnknownColumns(const ColumnNames_t &requiredCols, const Column
 
 bool IsInternalColumn(std::string_view colName);
 
+/// Returns the list of Filters defined in the whole graph
+std::vector<std::string> GetFilterNames(const std::shared_ptr<RLoopManager>& loopManager);
+
+/// Returns the list of Filters defined in the branch
+template <typename NodeType>
+std::vector<std::string> GetFilterNames(const NodeType& node)
```

Comment:
`std::shared_ptr<NodeType>` to be a bit more conservative in the amount of types that are matched here?

Reply:
It's not exposed to the user, do you think it would be better to have overloads accepting each one a node tye?

Reply:
it's exposed to us :) more specific signature is always better.

i wouldn't have overloads for each of the node types because they would all do exactly the same thing. when #2182 is in maybe we can just take a `RNodeBase`.